### PR TITLE
Adding consultant reference page

### PR DESCRIPTION
**Pull request by melcorr**
I created this page as a reference that we can use to direct community members who request professional help to.
Feedback welcome.

### CODE REVIEW

File: professional-support.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The following companies provide Foreman consulting services. If you would like to avail of their services, please contact them directly.
+
+### ATIX AG
+
+For more information, check out their [Foreman](https://atix.de/en/theforeman/#) consulting page. You can also about ATIX's areas of focus and contributions to the wider Foreman community in the guest blog [ATIX in the Foreman Community](https://theforeman.org/2020/10/atix-in-the-foreman-community.html)
+
+### iRonin.IT
+
+For more information, check out their [Foreman](https://www.ironin.it/foreman-development-services.html?utm_source=mailing&utm_medium=email&utm_campaign=foreman&utm_term=october) consulting page. You can read about iRonin's contributions to Foreman in the iRonin guest blog [iRonin in the Foreman Community](https://theforeman.org/2020/09/ironin-in-the-foreman-community.html).
```
Comment (tbrisker) [inline professional-support.md]:
```suggestion
For more information, check out their [Foreman](https://www.ironin.it/foreman-development-services.html) consulting page. You can read about iRonin's contributions to Foreman in the iRonin guest blog [iRonin in the Foreman Community](https://theforeman.org/2020/09/ironin-in-the-foreman-community.html).
```
Maybe also mention that they specialize in custom development for foreman, and the others are more of support/operation consulting?

File: professional-support.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+layout: professional-support
+title: Professional Support
+---
+
+# Professional Support
```
Comment (tbrisker) [inline professional-support.md]: Perhaps this page should be `Professional Services` as it isn't only support?

File: professional-support.md (hunk 1 above)
Comment (melcorr) [reply professional-support.md]: ty

**Review by lzap — APPROVED** [review]

**Comment by lzap** [comment]
Thank you!

**Pull request merged**